build_tools/azure/posix-docker.yml

```diff
@@ -58,27 +59,26 @@ jobs:
     # the JUNITXML file
     - script: >
         docker container run --rm
-        --volume $TEST_DIR:/temp_dir
+        --volume $TEST_DIR:$TEST_DIR
```

Comment:
Although, I do not see any issues with this, I think it is uncommon to map the whole parent directory structure into the Docker image. My preference is to stick with `/temp_dir` and `/ccache` to be consistent with how `/io` is mapped.

Reply:
OK I reverted this part